docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,126 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+<box type="tip" seamless>
+The list icons can be customized by adding an {icon="..."} at the end of a list item or a list itself.
+</box>
+
```

Comment:
Try to use examples that looks like realistic usage. We don't want the reader to look at an example and think "Why would I want to do _that_?"